ibis/backends/duckdb/__init__.py

```diff
@@ -45,6 +45,15 @@ def normalize_filenames(source_list):
     return list(map(util.normalize_filename, source_list))
 
 
+def _compile_temp_view(table_name, source):
+    from duckdb_engine import Dialect as DuckDBDialect
+
+    raw_source = source.compile(
+        dialect=DuckDBDialect(), compile_kwargs=dict(literal_binds=True)
```

Comment:
If this was a method on the backend instead of a standalone function, you could reuse the dialect at `self.con.dialect`. Not sure how much that matters, just a note.

Reply:
Oh, right. I think that makes more sense.

Reply:
Done.